src/ui/common/src/components/layouts/NavBar.tsx

```diff
@@ -118,7 +117,7 @@ const NavBar: React.FC<{
               );
             }
             return (
-              <Link
+              <RouterLink
```

Comment:
Did this work? Curious if this helps disabling the refresh when you click on the breadcrumb link

Reply:
This is how it looks:

https://github.com/aqueducthq/aqueduct/assets/30596854/e37c5036-4426-4b50-a2b3-bc62db672bb2

I wasn't too sure what you meant by disabling the refresh.

Reply:
Oh I meant did you see lots of API call in your server log when you click on the link? Previously it does as we refetch these APIs, which didn't utilize the caching power of RTK framework

Reply:
It's not a big deal if it doesn't fix. We can file a task and investigate later

Reply:
I think it does. When clicking "workflows" from the workflow detail page of workflow 0, only the details of the workflow 0 was filled in. The rest had to be loaded in from the server.

Reply:
Hmm okay let's file a separate task then